test/unit/specs/ImageCards.spec.js

```diff
@@ -1,25 +1,106 @@
 import Vue from 'vue'
 import sinon from 'sinon'
 import ImageCards from '@/components/ImageCards'
+import Vuex from 'vuex'
+import slide from '@/store/modules/slide/index'
+import { SET } from '@/store/modules/slide/mutation-types'
 
-describe('ImageCards.vue', () => {
-  it('can render correct contents', () => {
-    const Constructor = Vue.extend(ImageCards)
-    const vm = new Constructor().$mount()
-    expect(vm.cards.length)
-      .to.equal(7)
-  })
+describe('ImageCards.vue', function () {
+  let Constructor, vm, mockCurrentSlide
 
-  it('correctly emits a delete image call', () => {
-    const Constructor = Vue.extend(ImageCards)
-    const vm = new Constructor().$mount()
+  before(function () {
+    Vue.use(Vuex)
+    Constructor = Vue.extend(ImageCards)
+    const store = new Vuex.Store(slide)
+    vm = new Constructor({ store }).$mount()
     sinon.spy(vm, '$emit')
+    mockCurrentSlide = {
+      images: [
+        {
+          src: 'https://picsum.photos/200/300/?image=92'
```

Comment:
Are these links ever going to expire?

Reply:
We hope not, at least for the next two months https://picsum.photos/